Unity-Technologies/ScriptableRenderPipeline
# Fixed issue that caused GLES3 to render darker in GLES3 and Linear

### Purpose of this PR
 Fix issue that caused GLES3 to render darker in GLES3 and Linear
https://fogbugz.unity3d.com/f/cases/1169789/

---
### Release Notes
 Added

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- [x ] C# and shader warnings (supress shader cache to see them)
- Other: 

**Automated Tests**: Can't add automated tests for android atm.

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=universal/bugfix/1169789&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: Low. Few lines. Changes isolated to blit pass.

**Halo Effect**: None.

---
### Comments to reviewers
 I tested on a S5 on master and didn't reproduce the issue, but regardless of that the final blit was wrong.
